divide_and_conquer/quickhull.py

```diff
@@ -0,0 +1,89 @@
+'''
+Quick-hull-
+Created on 22/7/2020
+---description---
+Quick-hull is a divide and conquer algorithm designed to construct a convex hull.
+it is called Quick-hull as its logic is closer to Quick-sort.
+Complexity: O(n**2)

```

Comment:
```suggestion
Quickhull
Created on 22/7/2020
---description---
Quickhull is a divide and conquer algorithm designed to construct a convex hull.
The algorithm is called Quickhull because its logic is closer to Quicksort.
Complexity: O(n**2)
See: https://en.wikipedia.org/wiki/Quickhull
```